src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
-export interface Figure {
-
+interface Figure {
+  shape: string,
+  color: string,
```

Comment:
Make enums or union types here, to avoid using bare strings
```suggestion
  shape: Shape,
  color: Color,
```